scripts/Dockerfile

```diff
@@ -17,13 +17,7 @@
 
 From fedora:27
 
-RUN dnf install -y qemu-img parted gdisk make gcc bc git e2fsprogs
+RUN dnf install -y qemu-img parted gdisk make gcc bc git e2fsprogs go libudev-devel
```

Comment:
do you know what version of go is installed?

Reply:
I'dont I can installed from golang compiled binaries to have a version we know it works.

Reply:
Good point @devimc. I agree @jcvenegas - that is going to be the safest approach.